pep-0678.rst

```diff
@@ -330,6 +343,17 @@ does not address the original motivation of compatibility with
 Furthermore, we do not believe that the problem we are solving requires or
 justifies new language syntax.
 
+
+Acknowledgements
+================
+We wish to thank the many people who have assisted us through conversation,
+code review, design advice, and implementation: Adam Turner, Alex Grönholm,
+André Roberge, Barry Warsaw, Brett Cannon, CAM Gerlach, Carol Willing, Damian,
+Erlend Aasland, Gregory Smith, Guido van Rossum, Irit Katriel, Jelle Zijlstra,
+Ken Jin, Kumar Aditya, Mark Shannon, Matti Picus, Petr Viktorin, Zac
+Hatfield-Dodds, and psudeonymous commenters on Discord and Reddit.
```

Comment:
Maybe remove yourself because you're the one doing the thanking.

Reply:
```suggestion
Hatfield-Dodds, and pseudonymous commenters on Discord and Reddit.
```